packages/components/src/components/as-category-widget/as-category-widget.scss

```diff
@@ -12,12 +12,10 @@ as-category-widget {
   display: block;
   background: var(--category-widget--background-color, $color-ui-01);
 
-  .as-category-widget__heading {
-    margin: 0;
-    font: $font-subheader;
+  .content {
+    min-height: 100px;
```

Comment:
Why this particular height? Based on the sketch files?

Reply:
totally random 😂